src/javascript/app/pages/resources/metatrader/download.ui.js

```diff
@@ -0,0 +1,48 @@
+import OSDetect from '../../../../_common/os_detect';
+
+const toggleDownloadPage = target => {
+    document.querySelectorAll('.download-block').forEach(block => {
+        if (block.getAttribute('id') === target) {
+            block.setVisibility(1);
+        } else {
+            block.setVisibility(0);
+        }
```

Comment:
why not `block.setVisibility(block.getAttribute('id') === target)`?

Reply:
I'm personally against one-liners. I think one level `if` statements are more readable, especially when it comes to doing something based on the result if the statement. Although using a short-hand version is fine in this case too (and I am going to use it), It is not future proof as adding more logic, later on, will make the code more ugly.